### PR TITLE
Tabby Cat (combined)

### PR DESCRIPTION
**Pull request by ljmwaugh**
This is the combined Tabby Cat PRs from Red Hat Summit 2024.

This superseeds #962 #1060

### CODE REVIEW

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Review by bjhargrave — CHANGES_REQUESTED** [review]

File: knowledge/animals/tabby_cat/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      a mix of orange and white, the ratio between fur color varies,
+      from a few orange spots on the back of a white cat to a
+      completely orange coloring with no white at all.
+      The orange areas can be darker or lighter spots or stripes,
+      but the white is nearly always solid and usually appears on
+      the underbelly, paws, chest, and muzzle.
+   question: What is an orange tabby?
+task_description: 'this is knowledge contribution about tabby cats'
+document:
+ repo: https://github.com/ljmwaugh/instructlab_knowledge.git
```
[the remark this answers is not in the document]
Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml]: changed to use juliadenham

File: knowledge/animals/tabby_cat/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      The orange tabby, also commonly called red or ginger tabby,
+      is a color-variant of the above patterns, having pheomelanin
+      (O allele) instead of eumelanin (o allele). Though generally
+      a mix of orange and white, the ratio between fur color varies,
+      from a few orange spots on the back of a white cat to a
+      completely orange coloring with no white at all.
+      The orange areas can be darker or lighter spots or stripes,
+      but the white is nearly always solid and usually appears on
+      the underbelly, paws, chest, and muzzle.
+   question: What is an orange tabby?
```
[the remark this answers is not in the document]
Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml:61]: removed the second question

File: knowledge/animals/tabby_cat/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      or blotched, ticked, and spotted tabby patterns.
+   question: how can you tell a cat is a tabby cat
+ - answer: |
+      The English term tabby originally referred to "striped silk taffeta",
+      from the French word tabis, meaning "a rich watered silk".
+   question: what is the origin of the name "tabby"
+ - answer: |
+      The four known distinct patterns, each having a sound genetic
+      explanation, are the mackerel, classic, ticked, and spotted tabby patterns
+   question: what are the patterns that tabby's have
```
[the remark this answers is not in the document]
Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml]: made suggested change

File: knowledge/animals/tabby_cat/qna.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      and tail, and characteristic striped, dotted, lined, flecked,
+      banded, or swirled patterns on the body: neck, shoulders, sides,
+      flanks, chest, and abdomen. The four known distinct patterns,
+      each having a sound genetic explanation, are the mackerel, classic
+      or blotched, ticked, and spotted tabby patterns.
+   question: how can you tell a cat is a tabby cat
+ - answer: |
+      The English term tabby originally referred to "striped silk taffeta",
+      from the French word tabis, meaning "a rich watered silk".
+   question: what is the origin of the name "tabby"
```
[the remark this answers is not in the document]
Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml]: made suggested change

File: knowledge/animals/tabby_cat/qna.yaml [hunk 5]
```diff
@@ -0,0 +1,69 @@
+created_by: sborenst,jregehr
+version: 2
+domain: tabby cat
+seed_examples:
+ - answer: domestic cat
+   question: what kind of cat are tabby
+ - answer: |
+      distinctive M-shaped marking on their forehead, stripes by their
```
[the remark this answers is not in the document]
Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml]: made suggested change

File: knowledge/animals/tabby_cat/qna.yaml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+      eyes and across their cheeks, along their back, around their legs
+      and tail, and characteristic striped, dotted, lined, flecked,
+      banded, or swirled patterns on the body: neck, shoulders, sides,
+      flanks, chest, and abdomen. The four known distinct patterns,
+      each having a sound genetic explanation, are the mackerel, classic
+      or blotched, ticked, and spotted tabby patterns.
+   question: how can you tell a cat is a tabby cat
```
[the remark this answers is not in the document]
Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml]: made suggested change

File: knowledge/animals/tabby_cat/qna.yaml [hunk 7]
```diff
@@ -0,0 +1,69 @@
+created_by: sborenst,jregehr
+version: 2
+domain: tabby cat
+seed_examples:
+ - answer: domestic cat
+   question: what kind of cat are tabby
```
[the remark this answers is not in the document]
Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml]: made suggested change

**Comment by bjhargrave** [comment]
@instructlab-bot precheck

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Comment by bjhargrave** [comment]
> Results can be found [here](https://instruct-lab-bot.s3.us-east-2.amazonaws.com/precheck-pr-1118-a1df9f9bf16f8c0ef62a19f58106d7e47923b3f4-job-396/index.html).

A number of questions had decent answers. Some did not. So once we update the commit for the knowledge document, we will do a generate check.

**Comment by bjhargrave** [comment]
@instructlab-bot generate

**Review by bjhargrave — APPROVED** [review]

**Review by bjhargrave — CHANGES_REQUESTED** [review]
This is the tabby cat PR. Federation square knowledge needs to be in its own PR. 

I think your issue is that you are not using a separate git branch for each PR. Generally you should never work on your main branch and submit PRs from your main branch.

**Comment by ljmwaugh** [comment]
@bjhargrave I've tried to clean it up.  Please let me know if I succeeded or if I need to do something additional.  I reverted the commit that added the federation_square.

**Comment by bjhargrave** [comment]
@Mergifyio squash

**Comment by bjhargrave** [comment]
I squashed the commits (which force pushed your main branch).

**Comment by ljmwaugh** [comment]
replaced by #1212

**Pull request closed without merging**